src/styles/Header.scss

```diff
@@ -20,6 +20,7 @@
             justify-content: flex-end;
             align-items: center;
             gap: 30px;
+            padding-right: 20px;
```

Comment:
is het hier niet beter `justify-content: center` te nemen? 
flex-end + padding-right werkt mekeer wat tegen

Reply:
padding right moet een beetje voor logout en register mooi uit te lijnen boven categorie